docs/conf.py

```diff
@@ -196,6 +196,7 @@ def rebuild_readme():
     "jupyter_execute",
     "notebooks/paper",
     "notebooks/HydroShare_integration.ipynb",
+    "notebooks-notworking",
```

Comment:
Shouldn't it already be excluded? If exclusion only works for folders, remove the line above.

Reply:
Ah true !  The new `notebooks-notworking` will already include this `HydroShare_integration.ipynb`.  Done in ab2a1a60c2c61fb167c9ebbb5bd7c2743ed155eb.